pkgs/desktops/gnome/extensions/systemd-manager/default.nix

```diff
@@ -0,0 +1,36 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "gnome-shell-extension-systemd-manager";
+  version = "16";
+
+  src = fetchFromGitHub {
```

Comment:
[gschemas.compiled](https://github.com/hardpixel/systemd-manager/blob/master/systemd-manager%40hardpixel.eu/schemas/gschemas.compiled) is platform-specific so you need to remove it and recompile it using `glib-compile-schemas`.